Changelog 1.4.0 (#146)
PnX-SI/widget-gtsi: CHANGELOG.md

# CHANGELOG

## 1.4.0 (2026-07-08)

- Use Taxref vernacular names first when available, then EUNIS Biodiversity Database, then Catalogue of Life, instead of just using Catalogue of Life vernacular names all the time (#145 by @amandine-sahl)

> - Add a text about how to contribute to GBIF in the footer (#137 by @jacquesfize)

Reviewer: ```suggestion
- [GBIF] Add an "How to contribute ?" note in the footer (#137 by @jacquesfize)
```

> - Don't display an error message if geolocation is not activated on the device (#139 by @jacquesfize)

Reviewer: ```suggestion
- Error messages are disabled when geolocation is refused by the user (#139 by @jacquesfize)
```

> - Don't display an error message and continue to request API when getting a 429 error (Too many requests) from an API (#142 by @amandine-sahl)

Reviewer: ```suggestion
- [GBIF] 429 (Too many requests) error message is disabled (#142 by @amandine-sahl)
```

> - Update Node.js 20 to 24 (by @jacquesfize)
> - Update dependencies (by @jacquesfize)

Reviewer: ```suggestion
- Update Node.js 20 to 24 (#141 by @jacquesfize)
- Update dependencies (#141 by @jacquesfize)
```
